titan/params/demographics.yml

```diff
@@ -297,12 +302,36 @@ demographics:
                 type: float
                 min: 0.0
                 max: 1.0
-              prob:
-                default: 0.0
-                description: "Probability that an agent in this class with HIV becomes enrolled in HAART at a given time step"
+              cap:
                 type: float
+                default: 0.0
+                description: "Percent of diagnosed agents on haart if haart cap is used"
                 min: 0.0
                 max: 1.0
+              enroll:
+                type: definition
+                description: "Probability that an agent in this class with HIV becomes enrolled in HAART at a given time step based on time since diagnosis (use 0 for haart cap)"
```

Comment:
clarify that the start and stop are not time steps, but time steps since diagnosis - a little unclear with current wording.  If 0 is not used with haart cap, is that bad? or is this param just ignored if haart.use_cap is true?